packages/core/src/extensions/clipboardTextSerializer.ts

```diff
@@ -1,12 +1,18 @@
+import { Fragment, Node, Slice } from '@tiptap/pm/model'
 import { Plugin, PluginKey } from '@tiptap/pm/state'
+import { EditorView } from '@tiptap/pm/view'
 
+import { isCellSelection } from '../../../extension-table/src/utilities/isCellSelection.js'
 import { Extension } from '../Extension.js'
 import { getTextBetween } from '../helpers/getTextBetween.js'
 import { getTextSerializersFromSchema } from '../helpers/getTextSerializersFromSchema.js'
 
 export type ClipboardTextSerializerOptions = {
   blockSeparator?: string,
 }
+interface _Fragment extends Fragment {
+  content: Node[];
+}
```

Comment:
I don't think we should be extending prosemirror types. If it says it does not have a content property then it probably is not meant to be accessed

Reply:
> I don't think we should be extending prosemirror types. If it says it does not have a content property then it probably is not meant to be accessed

However, there is indeed a content property, and the type of this object is _Fragment, with its parent being _Slice. I couldn't find where they originated from; neither tipTap nor ProseMirror defines them. So I added _Fragment myself. Do you have any good solutions to

Reply:
content is not meant to exposed though because it is an internal property in prosemirror: https://github.com/ProseMirror/prosemirror-model/blob/cde085e345b6815c54af9c386feb73bce3ad41ee/src/fragment.ts#L17-L18

Reply:
There is a forEach method which allows iterating over each item: https://github.com/ProseMirror/prosemirror-model/blob/cde085e345b6815c54af9c386feb73bce3ad41ee/src/fragment.ts#L168

But I'm not sure about this approach in general

Reply:
> There is a forEach method which allows iterating over each item: https://github.com/ProseMirror/prosemirror-model/blob/cde085e345b6815c54af9c386feb73bce3ad41ee/src/fragment.ts#L168
> 
> But I'm not sure about this approach in general

Thank you for your suggestion. I'll give it a try later.